modules/ROOT/pages/migrate-mule3-apimanager.adoc

```diff
@@ -7,34 +7,36 @@ You migrate your existing Mule 3 applications that are managed by API Manager to
 
 . <<prepare-migration,Prepare for your migration>>
 +
-Based on how your existing Mule 3 application is configured, carefully review the prerequisites and determine how to plan the migration.
-. <<determine-migration-approach,Determine the migration approach>>
+Based on how your existing Mule 3 application is configured, carefully review the prerequisites to prepare for your migration.
+. <<determine-migration-approach,Determine your migration approach>>
 +
-Based on several factors, such as whether the migration requires a downtime, requires removing applied policies, or requires migrating contracts, you use the existing API instance or create an instance for migration.
+Based on several factors, such as whether the migration requires taking downtime, removing applied policies, or requires migrating contracts, either use an existing API instance or create an instance for the migration.
```

Comment:
Still an issue with parallel list items:
```suggestion
Based on several factors, such as whether the migration requires 
taking downtime, 
removing applied policies, 
or migrating contracts, 
either use an existing API instance or create an instance for the migration.
```
(broken into lines to illustrate list items)